packages/uikit/src/components/RoiCalculatorModal/RoiCalculatorFooter.tsx

```diff
@@ -65,12 +67,18 @@ const RoiCalculatorFooter: React.FC<React.PropsWithChildren<RoiCalculatorFooterP
   } = useTooltip(
     <>
       <Text>
-        {t(
-          "The Multiplier represents the proportion of CAKE rewards each farm receives, as a proportion of the CAKE produced each block."
-        )}
+        {rewardCakePerSecond
```

Comment:
Hmm. Could we avoid Aptos or BSC logic inside `packages/uikit` ? `packages/uikit` should not know about this logic to be generic